torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7182,17 +7182,26 @@ def error_inputs_kthvalue(op_info, device, **kwargs):
             ErrorInput(SampleInput(torch.tensor(2, device=device), args=(3,)),
                        error_type=RuntimeError, error_regex=k_out_of_range_err),)
 
+def sample_inputs_dropout(op_info, device, dtype, requires_grad, *,
+                          train=None, min_input_dim=None, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
-def sample_inputs_dropout(op_info, device, dtype, requires_grad, **kwargs):
-    make_arg = partial(make_tensor, (S,), device=device, dtype=dtype, requires_grad=requires_grad)
+    if min_input_dim:
+        # Create cases with dim ranging from min_input_dim to min_input_dim + 2 (inclusive)
+        cases = [(S,) * i for i in range(min_input_dim, min_input_dim + 3)]
```

Comment:
This is to generalize cases when the minimum input dimension supported is given (`min_input_dim`). For example, `feature_alpha_dropout` supports minimum 2D inputs, hence, the `cases` generated will be:

```
>>> cases = [(S,) * i for i in range(min_input_dim, min_input_dim + 3)]
>>> cases
[(5, 5), (5, 5, 5), (5, 5, 5, 5)]
```